app/views/membership.html

```diff
@@ -0,0 +1,292 @@
+<project-header class="top-header"></project-header>
+<project-page class="membership" ng-if="project">
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+        </div>
+      </div>
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>
+                Membership
+                <small class="pull-right meta">
+                  <a href="{{'roles' | helpLink}}">
+                    Learn more <i class="fa fa-external-link"></i>
+                  </a>
+                </small>
+              </h1>
+              <alerts alerts="alerts"></alerts>
+            </div>
+          </div>
+          <div  ng-if="!('rolebindings' | canI : 'list')">
+            <p>Nothing to see here, move along folks.</p>
+          </div>
+          <uib-tabset
+            ng-if="'rolebindings' | canI : 'list'">
+            <uib-tab
+              ng-repeat="(subjectKindName, subjectKind) in subjectKindsForUI | orderBy: 'sortOrder'"
+              active="selectedTab[subjectKind.name]"
+              select="selectTab(subjectKind.name)">
+              <uib-tab-heading>
+                {{subjectKind.name | startCase}}s&nbsp;({{subjectKind.subjects | hashSize}})
+              </uib-tab-heading>
+              <div ng-if="subjectKind.description">
+                <p>
+                  {{subjectKind.description}}
+                  <a
+                    ng-if="subjectKind.helpLinkKey"
+                    target="_blank"
+                    href="{{subjectKind.helpLinkKey | helpLink}}"
+                    class="meta">
+                    Learn more <i class="fa fa-external-link"></i>
+                  </a>
+                </p>
+              </div>
+              <div column class="content-pane content-{{subjectKind.name.toLowerCase()}}">
+
+                <div
+                  class="col-heading item-row"
+                  row mobile="column" tablet="column">
+                  <div row flex>
+                    <div class="col-name">
+                      <h3>Name</h3>
+                    </div>
+                    <div class="col-roles" flex>
+                      <h3>Roles</h3>
+                    </div>
+                  </div>
+                  <div class="col-add-role">
+                    <h3
+                      ng-show="mode.edit"
+                      class="hidden-xs">
+                      Add another role
+                    </h3>
+                    <a
+                      href=""
+                      ng-if="'rolebindings' | canI : 'update'"
+                      ng-show="!(mode.edit)"
+                      ng-click="mode.edit = !mode.edit">
+                      Edit {{ (subjectKind.name | startCase).toLowerCase() }} membership
```

Comment:
will the humanizeKind filter do what you need here? it already handles the startCase and the lower case for you (if you pass true as a second param it won't do the lower case).  I see this in a couple places in this file